docs/syntax-reference.rst

```diff
@@ -885,6 +893,62 @@ they are explicitly terminated.
 
 (To terminate a non-persistent query use ``Ctrl-C`` in the CLI.)
 
+.. _operators:
+
+=========
+Operators
+=========
+
+KSQL supports the following operators in value expressions:
+
+The explanation for each operator includes a supporting example based on the following table:
+
+.. code:: sql
+
+  CREATE TABLE USERS (
+      USERID BIGINT
+      FIRST_NAME STRING,
+      LAST_NAME STRING,
+      NICKNAMES ARRAY<STRING>,
+      ADDRESS STRUCT<STREET_NAME STRING, NUMBER INTEGER>
```

Comment:
maybe a nit but this struct example could be improved - generally you'd have the building number come first and it would be a string, not int (think "22B Baker St" for example) and likely you'd have a city field too. Just reduce the cognitive dissonance of reading this by making the things that should be familiar conform to the reader's expectation as far as possible